Apps/phsmime/readme.html

```diff
@@ -0,0 +1,35 @@
+<!-- File: readme.html
+  Copyright (c) 2020 Splunk Inc.
+
+  SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
+  without a valid written license from Splunk Inc. is PROHIBITED.
+-->
+
+<html>
+  <head></head>
+  <body>
+
+    <h2><a id="Requirements_2"></a>Requirements</h2>
+    <p>Supported OS is CentOS 7+</p>
+    <p>Access your Phantom instance via Terminal and install following OS dependencies</p>
+    <pre><code class="language-bash">
+    <span class="hljs-comment"># Install library and its dependencies</span>
+    $~ sudo yum install python-devel openssl-devel gcc gcc-c++
+    </code></pre>
+
+    <h2><a id="Keys_and_Certificates_3"></a>Keys and Certificates</h2>
+    <p>To use this app you need to provide an RSA key pair (this consists of a
+      public key and a private key) and an X.509 certificate of said public key.</p>
+    <p>To generate new RSA key pair, you could install and use OpenSSL on a CentOS 7
+      instance. Once properly installed, execute the following commands
+      to create key pairs and certificates.</p>
+    <pre><code class="language-bash">
+    <span class="hljs-comment"># Create a pair of keys</span>
+    $~ openssl req -newkey rsa:1024 -nodes -x509 -days 365 -out cert.pem
```

Comment:
`openssl req -newkey rsa:1024 -nodes -x509 -days 365 -out cert.pem`

Is there any specific reason for using a 1024 bit RSA key? We will like to suggest that you should use at least a 2048 bit RSA key as an example.